src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -170,7 +209,7 @@ public class MethodologyService : IMethodologyService
                 .OnSuccess(_ => GetSummary(id));
         }
 
-        private async Task<MethodologyVersionSummaryViewModel> BuildMethodologySummaryViewModel(
+        private async Task<MethodologyVersionViewModel> BuildMethodologySummaryViewModel(
```

Comment:
We should rename this method to `BuildMethodologyVersionViewModel`

Reply:
Thanks, done this renaming in 2027c7b9a00eb143fd485c50b0786cab239974a4 and the same applies to all the other comments left about naming.